extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -54,6 +54,7 @@ export class LiveListInterface {
    * Update the underlying live list dom structure.
    *
    * @param {?Element} element
+   * @return {time}
```

Comment:
I don't think so. I think I originally used number and reviewed a PR from @dvoytenko using time and switched over.